source/utils/CMakeLists.txt

```diff
@@ -1,10 +1,13 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: MIT
 
-add_library(utils
-  STATIC
-  "logging.h"
-  "logging.cpp"
-)
+set(logging_SOURCES logging.h logging.cpp)
```

Comment:
It is preferred to not use patterns like `set(*_SOURCES)`, etc. These are mostly patterns from autotools and it gives the impression that the CMake project is not a native CMake project.

Reply:
I've amended it to logging_files if that fits better?

Reply:
The usual pattern is `add_library` and `target_sources` but this is still an improvement